diff --git a/src/src/utils/unionimage.cpp b/src/src/utils/unionimage.cpp
index d4fe757..6665c71 100644
--- a/src/src/utils/unionimage.cpp
+++ b/src/src/utils/unionimage.cpp
@@ -144,8 +144,8 @@ public:
         m_freeimage_formats["DDS"]     =  FIF_DDS;
         m_freeimage_formats["GIF"]     =  FIF_GIF;
         //m_freeimage_formats["HDR"]     =  FIF_HDR;//FHDR covert failed
-        m_freeimage_formats["FAX"]     =  FIF_FAXG3;
-        m_freeimage_formats["G3"]      =  FIF_FAXG3;//FAXG3
+        m_freeimage_formats["FAX"]     =  FIF_TARGA;
+        m_freeimage_formats["G3"]      =  FIF_TARGA;//FAXG3
         m_freeimage_formats["SGI"]     =  FIF_SGI;
         m_freeimage_formats["EXR"]     =  FIF_EXR;
         //m_freeimage_formats["J2K"]     =  FIF_J2K;
